packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -0,0 +1,54 @@
+/********************************************************************************
+ * Copyright (C) 2018 Bitsler and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { interfaces } from 'inversify';
+import { createPreferenceProxy, PreferenceProxy, PreferenceService, PreferenceContribution, PreferenceSchema } from '@theia/core/lib/browser';
+
+export const TerminalConfigSchema: PreferenceSchema = {
+    type: 'object',
+    properties: {
+        'terminal.enableCopy': {
+            type: 'boolean',
+            description: 'Enable ctrl-c (cmd-c on macOS) to copy selected text',
+            default: false
```

Comment:
Note that I wouldn't be against setting these to true by default, since it's probably more common for people to use copy/paste than to use ctrl-v.

Reply:
+1 to set up true.

Reply:
If this gets merged, i'd be happy to have the default to true as well. +1

Reply:
Could you update it before merge?